cmd/convert_all/main.go

```diff
@@ -67,10 +69,13 @@ func newConMgr(ctx context.Context, cli *http.Client, host, srcBkt string, w int
 				}
 
 				fmt.Printf("Converting gs://%s/%s\n", srcBkt, obj)
-				if err := conReq(cli, host, obj, srcBkt); err != nil {
-					glog.Fatal(err)
+				err := backoff.Retry(func() error {
+					return conReq(cli, host, obj, srcBkt)
+				}, backoff.WithMaxRetries(backoff.NewConstantBackOff(30*time.Second), 3))
+				if err != nil {
+					glog.Errorf("failed to convert gs://%s/%s: %v", srcBkt, obj, err)
 				}
-				fmt.Printf("gs://%s/%s converted\n", srcBkt, obj)
+				time.Sleep(time.Second) // Avoid overloading the service.
```

Comment:
this looks like it'll delay 1s, not 30s.

Should this be:
  time.Sleep(30 * time.Second)

and maybe the 30 * time.Second is a const: "sleepTime" ?

Reply:
Good point. This is why I choose 1 qps: 
- Most of the time, 1 qps will not cause problems;
- occasionally, 1 qps can still hit a quota limit, so we will sleep 30 seconds and retry (which resolves all quota issues so far);
- anything > 1 qps can lead to much more quota issues;
- if each query takes 30s, the whole process will be too long (we have thousands of "month" directories).

I also recall seeing 1 qps for BQ data transfer API in some internal documents, but it's probably not accurate because I can still hit quota limit.

Good suggestion on the const - done.